packages/workflow/src/NativeMethods/Number.methods.ts

```diff
@@ -29,18 +29,61 @@ export const numberMethods: NativeDoc = {
 		toString: {
 			doc: {
 				name: 'toString',
-				description: 'Returns a string representing this number value.',
+				description:
+					'Converts the number to a simple textual representation. For more formatting options, see <code>toLocaleString()</code>.',
```

Comment:
I would keep `toString` because I expect the majority of usage will be without args (`num.toString()`). Maybe `Converts the number to a string representation`, the base argument is explained below. Will add some more simple examples.